src/core/references/translatingReferenceDetector.test.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+
+import {type NodeAddressT, NodeAddress} from "../graph";
+import type {URL, ReferenceDetector} from "./referenceDetector";
+import {MappedReferenceDetector} from "./mappedReferenceDetector";
+import {TranslatingReferenceDetector} from "./translatingReferenceDetector";
+
+const nodeA = NodeAddress.fromParts(["test", "A"]);
+const nodeB = NodeAddress.fromParts(["test", "B"]);
+
+function exampleDetector(pairs?: [URL, NodeAddressT][]): ReferenceDetector {
+  const map: Map<URL, NodeAddressT> = new Map(pairs);
+  const refs = new MappedReferenceDetector(map);
+  jest.spyOn(refs, "addressFromUrl");
+  return refs;
+}
+
+describe("core/references/translatingReferenceDetector", () => {
+  describe("TranslatingReferenceDetector", () => {
+    it("should implement the ReferenceDetector interface", () => {
```

Comment:
Since the TRD already is declared as implementing ReferenceDetector, I think this test is superfluous.